ckanext/switzerland/templates/scheming/package/resource_read.html

```diff
@@ -30,8 +30,95 @@ <h3>{{ _('Dataset description:') }}</h3>
   </div>
 {% endblock %}
 
-{%- block resource_more_items -%}
-{%- endblock -%}
+{%- set exclude_fields = [
+  'name',
+  'description',
+  'url',
+  'format',
+  ] -%}
+{%- set schema = h.scheming_get_dataset_schema(dataset_type) -%}
+
+{% block resource_additional_information_inner %}
+  {% if res.datastore_active %}
+    {% block resource_data_dictionary %}
+      {% if g.userobj %}
+        {% snippet 'datastore/snippets/dictionary_view.html',
+          ddict=h.datastore_dictionary(res.id), res=res %}
+      {% endif %}
+    {% endblock %}
+  {% endif %}
+
+  <div class="module-content">
+    {%- block additional_info_heading -%}<h2>{{ _('Additional Information') }}</h2>{%- endblock -%}
+    <table class="table table-striped table-bordered table-condensed" data-module="table-toggle-more">
+      {%- block additional_info_table_head -%}
+        <thead>
+          <tr>
+            <th scope="col">{{ _('Field') }}</th>
+            <th scope="col">{{ _('Value') }}</th>
+          </tr>
+        </thead>
+      {%- endblock -%}
+      <tbody>
+        {%- block resource_permalink -%}
+        <tr>
+          <th class="dataset-label">{{ _('Permalink') }}</th>
+          <td class="dataset-details">{% link_for _('Permalink'), named_route='ogdch_dataset.resource_permalink', id=pkg.name, filename=h.resource_filename(res.url) %}</td>
+        </tr>
+        {%- endblock -%}
+        {%- block resource_last_updated -%}
+        <tr>
+          <th scope="row">{{ _('Data last updated') }}</th>
+          <td>{{ h.render_datetime(res.last_modified) or h.render_datetime(res.created) or _('unknown') }}</td>
+        </tr>
+        {%- endblock -%}
+        {%- block resource_metadata_last_updated -%}
+        <tr>
+          <th scope="row">{{ _('Metadata last updated') }}</th>
+          <td>{{ h.render_datetime(res.metadata_modified) or h.render_datetime(res.created) or _('unknown') }}</td>
+        </tr>
+        {%- endblock -%}
+        {%- block resource_created -%}
+        <tr>
+          <th scope="row">{{ _('Created') }}</th>
+          <td>{{ h.render_datetime(res.created) or _('unknown') }}</td>
+        </tr>
+        {%- endblock -%}
+        {%- block resource_format -%}
+        <tr>
+          <th scope="row">{{ _('Format') }}</th>
+          <td>{{ res.format or res.mimetype_inner or res.mimetype or _('unknown') }}</td>
+        </tr>
+        {%- endblock -%}
+        {%- block resource_license -%}
+        <tr>
+          <th scope="row">{{ _('License') }}</th>
+          <td>{% snippet "snippets/license.html", pkg_dict=pkg, text_only=True %}</td>
+        </tr>
+        {%- endblock -%}
+        {%- block resource_fields -%}
+          {%- for field in schema.resource_fields -%}
```

Comment:
for the safety reason I would add a check before ,
to see if resource_fields exist and the doing `for loop`